discovery/src/main/java/io/airlift/discovery/client/Announcer.java

```diff
@@ -166,7 +167,13 @@ public void onFailure(Throwable t)
             {
                 Duration duration = errorBackOff.failed(t);
                 // todo this is a duplicate log message and should be remove after root cause of announcement delay is determined
```

Comment:
This logging was added in 633e4d5e85f0bfc49d722c6d024c870bf25ebc6c

@dain should we remove it, along with the one in `onSuccess`?

Reply:
Removed.